src/main/java/org/opentripplanner/transit/model/site/GroupStopBuilder.java

```diff
@@ -26,6 +27,11 @@ public class GroupStopBuilder extends AbstractEntityBuilder<GroupStop, GroupStop
     GeometryUtils.getGeometryFactory()
   );
 
+  private GeometryCollection encompassingAreaGeometry = new GeometryCollection(
+    null,
+    GeometryUtils.getGeometryFactory()
+  );
```

Comment:
This create new instances for each group stops, even for GTFS. I think this should eiter be a constant or nullable. If you use a constant the constant should be in the entity, not the builder. So, the easiest way to do it is to use `null`in the builder and set it to a constant in the entity - if builder value is `null`. But, I am happy with null as well - e.g. easier to se what is going on in a debugger.